KD_Lib/noisy/utils.py

```diff
@@ -1,24 +1,5 @@
 import torch
 
 
-def eval(model, data_loader):
-
-    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
-
-    model.eval()
-    total = len(data_loader.dataset)
-    correct = 0
-
-    with torch.no_grad():
-        for data, target in data_loader:
-            data, target = data.to(device), target.to(device)
-            output = model(data)
-            pred = output.argmax(dim=1, keepdim=True)
-            correct += pred.eq(target.view_as(pred)).sum().item()
-
-    print(f'{correct}/{total} correct')
-    print(f'The accuracy of the model is {correct/total}')
-
-
 def add_noise(x, variance=0.1):
     return x * (1 + (variance**0.5) * torch.randn_like(x))
```

Comment:
we can move this to the place where it is being used right? we don't need a new file just for this function.

Reply:
:+1: han, will do.